### ISSUE TITLE
Duplicate class found with FlexBox

### ISSUE THREAD

**Issue by H4zh4n** [Dhaval2404/ColorPicker#24]
If I have both implementations of `FlexBox ` and your  `ColorPicker` (like below) :

```
implementation 'com.github.Dhaval2404:ColorPicker:2.0'
implementation 'com.google.android.flexbox:flexbox:3.0.0'
```

I get this error when I run the project :

```
Execution failed for task ':app:checkDebugDuplicateClasses'.`
 A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
    Duplicate class com.google.android.flexbox.AlignContent found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.AlignItems found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.AlignSelf found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexContainer found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexDirection found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexItem found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexLine found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexWrap found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxHelper found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxHelper$1 found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxHelper$FlexLinesResult found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxHelper$Order found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxItemDecoration found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxLayout found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxLayout$DividerMode found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxLayout$LayoutParams found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxLayout$LayoutParams$1 found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxLayoutManager found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxLayoutManager$1 found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxLayoutManager$AnchorInfo found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxLayoutManager$LayoutParams found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxLayoutManager$LayoutParams$1 found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxLayoutManager$LayoutState found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxLayoutManager$SavedState found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.FlexboxLayoutManager$SavedState$1 found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
     Duplicate class com.google.android.flexbox.JustifyContent found in modules jetified-flexbox-2.0.1-runtime (com.google.android:flexbox:2.0.1) and jetified-flexbox-3.0.0-runtime (com.google.android.flexbox:flexbox:3.0.0)
```

Need some help on what to do please.

**Comment by H4zh4n** [comment]
I didn't mention why I have `ColorPicker:2.0` Instead of `ColorPicker:2.2`,
It is because I get below error with `2.2 `

```
Could not find com.github.Dhaval2404:ColorPicker:2.2.
Required by:
    project :app
```

OR  this upon sync :

`Failed to resolve: com.github.Dhaval2404:ColorPicker:2.2`

Same thing with `2.1` as I saw it was latest in releases, but `2.2` was written in README

**Comment by Dhaval2404** [comment]
@H4zh4n I messed up while releasing v2.1 and v2.2 updates. I will try to fix the issue by tomorrow.

**Comment by Dhaval2404** [comment]
@H4zh4n I have released v2.3. Please give it a try

**Comment by H4zh4n** [comment]
> @H4zh4n I have released v2.3. Please give it a try

Works perfectly, thanks a ton.
